topics/proteomics/tutorials/encyclopedia/tutorial.md

```diff
@@ -0,0 +1,428 @@
+>
+> 1. Create a new history for this tutorial
+> 2. Import the files from [Zenodo]({{ page.zenodo_link }}) or from
+>    the shared data library (`GTN - Material` -> `{{ page.topic_name }}`
+>     -> `{{ page.title }}`):
+>
+>    ```
+>    https://zenodo.org/record/4926594/files/Experimental_DIA_Dataset_Collection__191023JAT03_1_Mix_2_1ug_pOT_30k_390_1010_12%20.raw.raw
+>    https://zenodo.org/record/4926594/files/Experimental_DIA_Dataset_Collection__191023JAT10_2_Mix_4_1ug_pOT_30k_390_1010_12%20.raw.raw
+>    https://zenodo.org/record/4926594/files/Experimental_DIA_Dataset_Collection__191023JAT11_3_Mix_1_1ug_pOT_30k_390_1010_12%20.raw.raw
+>    https://zenodo.org/record/4926594/files/Experimental_DIA_Dataset_Collection__191023JAT12_4_Mix_3_1ug_pOT_30k_390_1010_12%20.raw.raw
+>    https://zenodo.org/record/4926594/files/GPF_DIA_Dataset_Collection__191023JAT04_P_1ug_395_505_4%20.raw.raw
+>    https://zenodo.org/record/4926594/files/GPF_DIA_Dataset_Collection__191023JAT05_P_1ug_495_605_4%20.raw.raw
+>    https://zenodo.org/record/4926594/files/GPF_DIA_Dataset_Collection__191023JAT06_P_1ug_595_705_4%20.raw.raw
+>    https://zenodo.org/record/4926594/files/GPF_DIA_Dataset_Collection__191023JAT07_P_1ug_695_805_4%20.raw.raw
+>    https://zenodo.org/record/4926594/files/GPF_DIA_Dataset_Collection__191023JAT08_P_1ug_795_905_4%20.raw.raw
+>    https://zenodo.org/record/4926594/files/GPF_DIA_Dataset_Collection__191023JAT09_P_1ug_895_1005_4%20.raw.raw
+>    https://zenodo.org/record/4926594/files/T4_Salmonella_Ecoli_Bacillus_BS_191102.fasta
+>    https://zenodo.org/record/4926594/files/T4_Salmonella_Ecoli_Bacillus_fasta_trypsin_z2_nce33_BS_191102.dlib
+>    ```
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>    {% snippet faqs/galaxy/datasets_import_from_data_library.md %}
+>
+> 3. For all the datasets that you have just uploaded, please rename them as follows:
+>     - The GPF files should be renamed as follows:
+>       - From: **GPF_DIA_Dataset_Collection__191023JAT06_P_1ug_595_705_4%20.raw.raw**
+>       - To: **191023JAT06_P_1ug_595_705_4.raw**
+>       - Note that the "%20" characters were removed.
+>     - The Experimental design files should be renamed as follows:
+>       - From: **Experimental_DIA_Dataset_Collection__191023JAT12_4_Mix_3_1ug_pOT_30k_390_1010_12%20.raw.raw**
+>       - To: **191023JAT12_4_Mix_3_1ug_pOT_30k_390_1010_12.raw**
+>       - Again, the "%20" characters were removed
+>     - The Background FASTA file will be renamed similarly:
+>       - From: **T4_Salmonella_Ecoli_Bacillus_BS_191102.fasta**
+>       - To: **T4_Salmonella_Ecoli_Bacillus_BS_191102.fasta**
+>     - Lastly, the DLIB library is renamed:
+>       - From: **T4_Salmonella_Ecoli_Bacillus_fasta_trypsin_z2_nce33_BS_191102.dlib**
+>       - To: **T4_Salmonella_Ecoli_Bacillus_fasta_trypsin_z2_nce33_BS_191102.dlib**
+> 4. Check that the datatype
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="datatypes" %}
```

Comment:
the last part of this snippet is only need if you want to tell the user what exact datatype they should choose, something like:

```suggestion
>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="fasta" %}
```

But I think these files are all of different datatypes right? in that case I would remove the `datatype="datatypes"` part in this line, but describe more clearly in point 4 which files should be which datatype